### ISSUE TITLE
Reduce batches?

### ISSUE THREAD

**Issue by mataric** [isekaidev/stable.art#3]
Hey there, I run stable diffusion locally without any issue usually, however this plugin will automatically create images in batches of 4, which increases memory cost and causes me to get CUDA out of memory issues much more often. 

Would it be possible to get a slider for batch count added in, along with one for image resolution?

This plugin has the most potential I've seen to become perfect - thank you so much!

**Comment by isekaidev** [comment]
About image resolution - for txt2img, you can use the rectangular marquee tool and specify a size you want.

About batch size - I do not want to add UI elements that will be used only once, so will try to fix it without sliders. How much VRAM do you have? I can get VRAM with API and then automatically decrease the batch size.

**Comment by mataric** [comment]
Ahh, I didn't realise that - that's a great way to do it.

I'm using a 12gb Geforce RTX 2060. I understand why you'd want to keep minimal usage UI off the page, but batch size is one I often change a lot. I just have to turn it down it down when generating large images or using inpainting. 

Perhaps a better option would be to create a second 'settings/options' page where settings like this, and 'Endpoint' could be adjusted once for the majority of people, but still tweaked by people like me who use them more?

**Comment by JTJutajoh** [comment]
I second the request for an option to customize the number of images generated. 4 images often isn't enough to cherrypick the good results from. Other times you might be doing a step with very low denoising to slightly clean up an image and only want to generate one at a time. I do not think that adding this to the UI would be unnecessary clutter. If that is a serious concern, it could be hidden under a collapsed section such as an "Advanced settings" section.

**Comment by uselessjunk** [comment]
Same here, a lower batch count would allow us to generate larger images / areas.
If I select a large area and run out of VRAM it freezes the generation process. If I then cancel it, it doesn't "clears" the VRAM.
It gets stuck at 24264MiB / 24576MiB and I need to restart anacaonda / the automatic111 GUI which slows down the workflow quite a bit.

**Comment by isekaidev** [comment]
Done 👍

<img width="565" alt="image" src="https://user-images.githubusercontent.com/118139897/213626554-b5664c84-0337-423c-8540-3408f87ebd12.png">

**Comment by remixer-dec** [comment]
It would be nice to have a number of images and a batch size configurable separately, to be able to generate multiple images on devices that do not have a lot of vRAM
